modules/features/shared/src/main/java/au/com/shiftyjelly/pocketcasts/shared/AppLifecycleObserver.kt

```diff
@@ -23,6 +25,7 @@ class AppLifecycleObserver @Inject constructor(
     override fun onResume(owner: LifecycleOwner) {
         super.onResume(owner)
         appLifecycleAnalytics.onApplicationEnterForeground()
+        FeatureFlagManager.refreshFeatureFlags()
```

Comment:
Might it be worth adding some kind of debounce here just to make sure we don't make this api call too much? Hopefully the remote feature flags won't be changing too often. 🤔

Reply:
I reduced the fetch interval to 1 sec in the test patch. We have a minimum fetch interval set to 2 hours [here](https://github.com/Automattic/pocket-casts-android/pull/1085/files#diff-68eeaa8c8ded9e538e37b89d78e81c210fe6a139510e555647ea1c10934c1f36R21). 

I think fetch on refresh should respect this interval. But let me check.

Reply:
It respects this interval on refresh. But I also had to activate remote config after fetch in 42fe398.